edenia/antelope-tools
# Fix/evm deploy adjustments

### Deploy adjustments

### What does this PR do?

- Remove secrets related with Telos evm from the workflows
- Avoid to count transactions from duplicated blocks
- Print a better error when a constraint violation occurs

### Steps to test

1. Check the .yaml files for Telos and Telos testnet
2. Check the correctness of the calculation of the total transactions.
